### PR TITLE
🐐 Show all time most paid channels in the YPP carrousel

### PR DESCRIPTION
**Pull request by thesan**
- #5124

Depends on:
- https://github.com/Joystream/orion/pull/222

### CODE REVIEW

File: packages/atlas/codegen.config.ts [hunk 1]
```diff
@@ -1,8 +1,13 @@
 import { CodegenConfig } from '@graphql-codegen/cli'
+import * as assert from 'assert'
 
 import { customSchemaLoader } from './scripts/customSchemaLoader'
 
-const schemaUrl = 'https://orion.gleev.xyz/graphql'
+const NODE_ENV = process.env.NODE_ENV?.toUpperCase() ?? 'PRODUCTION'
```
Comment (attemka) [inline packages/atlas/codegen.config.ts]: I think this should be either `VITE_ENV` or `VITE_DEFAULT_DATA_ENV`

File: packages/atlas/scripts/customSchemaLoader.ts [hunk 2]
```diff
@@ -17,30 +17,34 @@ async function fetchAuthCookie() {
   return response.headers['set-cookie']
 }
 
-export async function customSchemaLoader() {
-  const authCookie = await fetchAuthCookie()
-  const introspectionQuery = getIntrospectionQuery()
+export function customSchemaLoader(schemaUrl: string, authUrl: string) {
+  return async () => {
+    const authCookie = await fetchAuthCookie(authUrl)
```
Comment (attemka) [inline packages/atlas/scripts/customSchemaLoader.ts]: isn't the schema available without the auth cookie now?

**Review by attemka — CHANGES_REQUESTED** [review]

File: packages/atlas/scripts/customSchemaLoader.ts (hunk 2 above)
Comment (thesan) [reply packages/atlas/scripts/customSchemaLoader.ts]: Yeah probably actually I just copied the code I added on the notification branch a while ago

File: packages/atlas/codegen.config.ts (hunk 1 above)
Comment (thesan) [reply packages/atlas/codegen.config.ts]: Yes `VITE_DEFAULT_DATA_ENV` would make a lot of sense

**Review by attemka — APPROVED** [review]

**Pull request merged**